taker-frontend/src/components/Trade.tsx

```diff
@@ -222,14 +236,12 @@ export default function Trade({
                                     onClick={onOpen}
                                     h={16}
                                     w={"40"}
+                                    fontSize={"2xl"}
```

Comment:
Yeah - the previous button with the price was actually nice. Once thing that is missing as well, is the reference price (i.e. price without spread, that we displayed as index price before, but that is not what  is used internally, so that was actually not correct...).

I'd expect that with the perpetual position the spread will either go away or be minimal. Nevertheless, I think we need the current "closing"  price as known in the taker daemon - that would be the reference price for me; the fact that we are not using the index price there is an implementation detail of the daemon.
 The price that is currently displayed in the bubble is the maker's ask price, which is not enough. We definitely need bid once we add short, but we should also display the reference price again. 

Note that the reference price *might* be important when opening a position as well, because it shows the spread of the maker. If we assume that there is no spread then it is not important there. 
For closing the position the reference price is the *only* price that matters.

Reply:
Reduced it to `xl` - might change this with working on the prices